metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py

```diff
@@ -988,9 +1002,64 @@ def get_ownership(
     def _get_last_modified_time(
         self, looker_dashboard: Optional[LookerDashboard]
     ) -> Optional[datetime.datetime]:
-        if looker_dashboard is None:
+        return (
+            getattr(looker_dashboard, "last_updated_at", None)
```

Comment:
It should be safe to access `last_updated_at` here and `last_updated_at` few lines below, why `getattr`?

```
@dataclass
class LookerDashboard:
    id: str
    title: str
    ...
    last_updated_at: Optional[datetime.datetime] = None
    last_updated_by: Optional[LookerUser] = None
    deleted_at: Optional[datetime.datetime] = None
    deleted_by: Optional[LookerUser] = None
    ...
```